docs/paper/dev/api/pdc.md

```diff
@@ -104,15 +104,15 @@ The PDC supports a wide range of data types, such as:
   List<String> strings = container.get(key, PersistentDataType.LIST.strings());
   ```
 
-:::info[Boolean PersistentDataType]
+:::info
```

Comment:
```suggestion
:::info[Boolean PersistentDataType]
```

Reply:
since `PersistentDataType` is a class reference, I wanted to put it in backticks, however that resulted in it being snapped out of the admonition title into a separate element (you definitely can use backticks in those though, so I'm not sure what's the problem) - that's why I ended up removing it entirely

Reply:
![image](https://github.com/PaperMC/docs/assets/34477304/b3358ff0-9948-4e15-90d0-73d451807755)